dvc/progress.py

```diff
@@ -62,18 +65,30 @@ def __init__(
         desc  : persists after `close()`
         level  : effective logging level for determining `disable`;
             used only if `disable` is unspecified
+        disable  : If (default: None), will be determined by logging level.
+            May be overridden to `True` due to non-TTY status.
+            Skip override by specifying env var `DVC_ISATTY`.
         kwargs  : anything accepted by `tqdm.tqdm()`
         """
         kwargs = kwargs.copy()
         kwargs.setdefault("unit_scale", True)
+        kwargs.setdefault("file", sys.stderr)
         if bytes:
             bytes_defaults = dict(
                 unit="B", unit_scale=True, unit_divisor=1024, miniters=1
             )
             kwargs = merge(bytes_defaults, kwargs)
         self.desc_persist = desc
+        # auto-disable based on `logger.level`
         if disable is None:
             disable = logger.getEffectiveLevel() > level
+        # auto-disable based on TTY
+        if (
+            not disable
+            and DVC_ISATTY is None
```

Comment:
Empty string value of `DVC_ISATTY` should be same as not set.

Reply:
Also the name of the var is confusing. It doesn't rewrite isatty, but prevents its call. So, it's like `DVC_IGNORE_ISATTY`. 

I hate to mishmash it again )

Reply:
honestly I don't really care what it's called, both seem fine to me - @Suor would be great if you could discuss with @efiop what's more appropriate.

Reply:
`DVC_IGNORE_ISATTY` is fine with me. The issue I've had with the previous name is that it was starting with `TQDM_`.